.env.example

```diff
@@ -0,0 +1 @@
+GTAG_ID=<Google tag id>
```

Comment:
Can we please add table inside the `README.MD` where we describe all the env file, the config ecc that we have inside the repo? Something like this:
```
| Name    | Description | 
| ——————— | ——————————— |
| GTAG_ID | …..         | 
```

It's helpful for maintenance

Reply:
[c1fd3c4](https://github.com/osmosis-labs/osmosis-website/pull/60/commits/c1fd3c4eefe74e952a116ad835181eb7317337f2)